lineman/app/services/thread_query_service.coffee

```diff
@@ -52,13 +54,16 @@ angular.module('loomioApp').factory 'ThreadQueryService', (Records) ->
           view.applyFind(lastActivityAt: { $gt: moment().startOf('day').subtract(6, 'week').toDate() })
           view.applyWhere (thread) -> thread.isUnread()
 
-      if _.contains(filters, 'show_muted')
-        view.applyFind(volume: { $eq: 'mute' })
-      else
-        view.applyFind(volume: { $ne: 'mute' })
-
       _.each filters, (filter) ->
         switch filter
+          when 'show_muted'
+            view.applyFind($or: [{volume:  { $eq: 'mute' }},
```

Comment:
I don't think we can fuse them because we're serializing other people's memberships out as well, so it will get confusing.

I do think we can put 'volume' on the group serializer, plus making the 'volume' method on threads a little bit smarter

```
# discussion_model.coffee
threadVolume: ->
  @volume or @group().volume
```